config.json

```diff
@@ -22,48 +22,47 @@
       ]
     },
     {
-      "slug": "leap",
-      "uuid": "66d974b5-18fc-4993-b5f2-7beda4f4afa3",
+      "slug": "two-fer",
+      "uuid": "2ee3cc7a-db3f-4668-9983-ed6d0fea95d1",
       "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-        "control_flow_if_else_statements",
-        "integers"
+        "optional_values",
+        "strings"
       ]
     },
     {
-      "slug": "bob",
-      "uuid": "b3c4d578-10c8-47bc-b0ae-149ed8da530a",
+      "slug": "leap",
+      "uuid": "66d974b5-18fc-4993-b5f2-7beda4f4afa3",
       "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
         "control_flow_if_else_statements",
-        "strings"
+        "integers"
       ]
     },
     {
-      "slug": "sum-of-multiples",
-      "uuid": "e702b75e-4c9e-40ef-bcb1-674a87222c23",
+      "slug": "accumulate",
```

Comment:
I both like and fear having this exercise early. The thing I like is that it introduces a concept (recursion) that is vital to the track. The implementation itself is also quite simple, but there is a nasty minor detail (tail recursion) hiding in there. We could perhaps instruct mentors to not mention tail recursion, although this is the only exercise we have that teaches this I think.

The thing I'm concerned about, is that the concept and it's application might be difficult for people to grasp. Also, the stub definition introduces generic types (`let accumulate (func: 'a -> 'b) (input: 'a list): 'b list = ...`), so that could be a bit daunting to read. This could all very well not be true, and I wouldn't want to put it after `space-age`, as that exercise is slightly more involved to solve. So the remaining option is to swap it with `raindrops`, of which I'm not sure that improves the situation a lot. For now, let's keep it here and check regularly to see if this becomes a new blocking point. Maybe people have less problems with this than I anticipate.

Reply:
Let's keep a very sharp eye on this one. I think it only moved forward this much because of the extremely low wait time on it. If you think it's worth swapping it with raindrops now, let's do that manually in this PR!